octoprint_mrbeam/__init__.py

```diff
@@ -2117,7 +2127,7 @@ def on_api_command(self, command, data):
                         parse_csv(
                             device_model=self.get_model_id(),
                             laserhead_model=self.get_current_laser_head_model(),
-                            laser_cutter_mode=self.get_laser_cutter_mode(),
+                            laser_cutter_mode = self.get_laser_cutter_mode() if self.get_laser_cutter_mode() is not None else LaserCutterModeEnum.DEFAULT.value,
```

Comment:
Suggestion (no need to change it if you prefer the other way):
```suggestion
                            laser_cutter_mode = self.get_laser_cutter_mode() or LaserCutterModeEnum.DEFAULT.value,
```

Reply:
I will implement it.